lib/rucio/client/lifetimeclient.py

```diff
@@ -39,10 +39,38 @@ def list_exceptions(
             states: Optional['Sequence[LifetimeExceptionsState]'] = None
     ) -> 'Iterator[dict[str, Any]]':
         """
-        List exceptions to Lifetime Model.
-
-        :param id:         The id of the exception
-        :param states:     The states to filter
+        Lists lifetime model exceptions that allow extending data lifetimes beyond their configured policies.
+
+        The lifetime model exceptions are used to override the default lifecycle policies for data identifiers
+        (files, datasets, containers, or archives) that need to be kept longer than usual. These exceptions
+        can be filtered by their ID or approval state.
+
+        :param exception_id: The unique identifier of a specific exception. If provided, returns only that exception.
+        :param states: Filter exceptions by their states. Possible values are:
+                      - `A` (APPROVED): Exception was approved
+                      - `R` (REJECTED): Exception was rejected
+                      - `W` (WAITING): Exception is waiting for approval
```

Comment:
Waiting approval from whom (what sort of admin approves them)? Useful information for troubleshooting.